hexa/data_collections/models.py

```diff
@@ -65,6 +65,9 @@ def delete_if_has_perm(
 
         self.delete()
 
+    def get_absolute_url(self) -> str:
```

Comment:
This will work but we typically use `reverse()` from `django.urls` for this.

Reply:
In this case we do not have urls for collections. Do we create stub urls for collections while we implement them in the new frontend ?

If not, what could I use as params in `reverse()` ?